### PR TITLE
Users/emawa/restart controlserver

### PR DESCRIPTION
**Pull request by awaemmanuel**

### CODE REVIEW

File: User Guides/19-Restarting a Control Server Deployment.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Restart a Failed Control Server
+The Control Server as the brain box of the entire system is still susceptible to failure. The following steps describes how to re-deploy a failed Control Server.
+
+#### Restart Control Server Deployment
+1. Take a note of your resource group name. 
+
+2. Login to [Azure](portal.azure.com)
```
Comment (ctstone) [inline User Guides/19-Restarting a Control Server Deployment.md]: Broken link. Add protocol

File: User Guides/19-Restarting a Control Server Deployment.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+The Control Server as the brain box of the entire system is still susceptible to failure. The following steps describes how to re-deploy a failed Control Server.
+
+#### Restart Control Server Deployment
+1. Take a note of your resource group name. 
+
+2. Login to [Azure](portal.azure.com)
+
+3. Locate your resource group
+
+4. Open the **Deployments** tab under your resource group.
```
Comment (ctstone) [inline User Guides/19-Restarting a Control Server Deployment.md]: Image displays inline with text. Add 2 spaces at end of this line to force markdown line break

File: User Guides/19-Restarting a Control Server Deployment.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    ![Deployment Tab](../img/restart_ctrl_server_assets/deployments.png)
+5. Click on **deployVMCtrlsv** 
+	![Click on Deploy Ctrl Server VM](../img/restart_ctrl_server_assets/deployments2.png)
+6. Click on **Redeploy** to re-provision the Control Server VM    
+	![Click on Deploy Ctrl Server VM](../img/restart_ctrl_server_assets/deployments3.png)
+7. This opens the **Custom Deployment**  
+	![Deployment Tab](../img/restart_ctrl_server_assets/deployments4.png)
```
Comment (ctstone) [inline User Guides/19-Restarting a Control Server Deployment.md]: This image is pretty low res and hard to read. Can it be made to match the others?

File: User Guides/19-Restarting a Control Server Deployment.md (hunk 2 above)
Comment (tyclintw) [reply User Guides/19-Restarting a Control Server Deployment.md]: Please do this for all images to force left align.

File: User Guides/19-Restarting a Control Server Deployment.md (hunk 1 above)
Comment (tyclintw) [reply User Guides/19-Restarting a Control Server Deployment.md]: Also, to be inline with the other docs please call it "Azure portal".

Comment (awaemmanuel) [reply User Guides/19-Restarting a Control Server Deployment.md]: Done

File: User Guides/19-Restarting a Control Server Deployment.md (hunk 2 above)
Comment (awaemmanuel) [reply User Guides/19-Restarting a Control Server Deployment.md]: Done

File: User Guides/19-Restarting a Control Server Deployment.md (hunk 3 above)
Comment (awaemmanuel) [reply User Guides/19-Restarting a Control Server Deployment.md]: Updated image

**Pull request merged**